.github/workflows/linters.yml

```diff
@@ -0,0 +1,45 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  eslint:
+    name: ESLint
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "18.x"
+      - name: Setup ESLint
```

Comment:
- [ ] You added linters for lint check. That's great 👍. But you did not add webhint for hint report. I suggest you to add .hintrc for web hint report. It will make your project more professional.